### DISCUSSION TITLE
biome formatter defaults to using tabs?

### DISCUSSION THREAD

**Discussion by jessekrubin** [biomejs/biome#4541] [General]
Curious why the biome formatter defaults to `tab` for `formatter.indentStyle` when prettier defaults to using spaces?

Not complaining just curious. 

I was under the impression that the biome formatter aimed to be close to prettier, and having to configure `indentStyle` conflicts with my personal formatter philosophy which is "dont configure the formatter -- use the defaults"

**Comment by ematipico** [comment]
There's a very old issue, where you can read more about the reasons: https://github.com/rome/tools/issues/425

Biome provides the majority of the same options available with Prettier, and we are a replacement in terms of formatting style and options, but we stand with our defaults that might be different.

As long as users can change the options, there shouldn't be debates. You can use `biome migrate prettier` to swiftly use Prettier defaults during migration

**Reply by jessekrubin** [reply]
@ematipico thanks! Just was curious. Stand by your defaults!

**Reply by jessekrubin** [reply]
"Stand by your defaults" (to the tune of 'stand by your man' by tammy wynette)

**Reply by musjj** [reply]
I feel that this should be mentioned in https://biomejs.dev/formatter/differences-with-prettier

**Reply by arturog** [reply]
> As long as users can change the options, there shouldn't be debates. You can use biome migrate prettier to swiftly use Prettier defaults during migration

Incredible this is still a thing in 2025.  Moving away -- good luck.
